WebARKit/WebARKitTrackers/WebARKitOpticalTracking/WebARKitTracker.cpp

```diff
@@ -400,7 +406,8 @@ class WebARKitTracker::WebARKitTrackerImpl {
         return featureMask;
     }
 
-    bool CanDetectNewFeatures() { return (_currentlyTrackedMarkers < _maxNumberOfMarkersToTrack); }
+    //bool CanDetectNewFeatures() { return (_currentlyTrackedMarkers < _maxNumberOfMarkersToTrack); }
```

Comment:
I had to override this because _maxNumberOfMarkersToTrack result = 0 !! so the boolean comparison is resulting always false and the detection routine never start. And _maxNumberOfMarkersToTrack is initialized to 1 so there is a bug somewhere....

Reply:
solved with commit https://github.com/webarkit/WebARKitLib/pull/16/commits/4e3ce3a66f5cf1068a5eeb4d446bd646b51cb13e